ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -149,8 +149,17 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-imdsv2
+- as: aws-ipi-minimal-permission-f28
```

Comment:
@jianlinliu Shall we use the short name 'mini-perm' as the others ?

Reply:
I will create another PR to update them for 4.12 later.